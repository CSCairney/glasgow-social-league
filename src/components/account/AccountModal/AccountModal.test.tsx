import React from "react";
import AccountsModal from "@/components/account/AccountModal/index";
import {renderWithProviders} from "@/helpers/test/test-utils";

describe("AccountModal", () => {
    const mockFunction = jest.fn();

    test("SNAPSHOT test for default layout", () => {
        const { asFragment } = renderWithProviders(
            <AccountsModal
                onCloseWithSession={mockFunction}
                onClose={mockFunction}
                sessionId={123}
                accounts={[]}
            />
        );
        expect(asFragment()).toMatchSnapshot();
    });
});
