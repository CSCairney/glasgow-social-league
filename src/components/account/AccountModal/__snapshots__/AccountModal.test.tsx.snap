// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountModal SNAPSHOT test for default layout 1`] = `
<DocumentFragment>
  <div
    class="modalOverlay"
  >
    <div
      class="modalContent"
    >
      <h2
        class="title"
      >
        Select Participants
      </h2>
      <div
        class="accountList"
      />
      <div
        class="modalActions"
      >
        <button
          class="startButton"
        >
          Start Session
        </button>
        <button
          class="closeButton"
        >
          Close
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
